examples/minimal/packages/client-react/src/mud/setupNetwork.ts

```diff
@@ -40,10 +40,23 @@ export async function setupNetwork() {
   const { components, latestBlock$, storedBlockLogs$, waitForTransaction } = await syncToRecs({
     world,
     config: mudConfig,
+    tables: {
```

Comment:
so `syncToRecs` syncs all tables passed `mudConfig` or in `tables` correct? What in `mudConfig` do we need in here besides the resolved tables? If it's only that, should we just pass a tables key (ie `syncToRecs({ tables: { ...mudConfig.tables, ...additionalTables } })`)? (mostly wondering about longer term when the config is resolved by default, fine to not do that here yet to avoid breaking changes)

Reply:
long term I think we should do that, yep! I just didn't want to encourage the intermediate step of using the experimental `resolveConfig` in user land